faq/faq.md

```diff
@@ -0,0 +1,351 @@
+software.
+
+If the license allows the recipient to [use, study, share and
+improve](https://fsfe.org/freesoftware/basics/4freedoms.html) the software, then
+that software is Free Software.
+
+## Which license should I choose? {#which-license}
+
+## Do I really need to include the license and copyright headers in all files? Why should I care? {#why-care}
+
+## What is SPDX? {#what-is-spdx}
+
+SPDX stands for [Software Package Data Exchange](https://spdx.org/). It is a
+project by the [Linux Foundation](https://www.linuxfoundation.org/) and the rock
+upon which REUSE is built. SPDX defines a standardized way to share copyright
+and licensing information between projects and people. Most importantly for
+REUSE, SPDX maintains the [SPDX License List](https://spdx.org/licenses/), which
+defines standardized identifiers for a lot of licenses.
+
+# The tool {#tool}
+
+## How do I install and use the REUSE tool? {#install-tool}
+
+The REUSE tool is a handy companion that can verify your compliance with REUSE
+and automate a lot of adjacent tasks. You can find the full documentation for
+the tool at <https://reuse.readthedocs.io/>. But the short of it is:
+
+```
+$ pip3 install --user fsfe-reuse 
+$ export PATH=~/.local/bin:$PATH
+$ reuse --help
+```
+
+## How do I exclude a file from REUSE compliance testing? {#exclude-file}
+
+If the file is a build artifact and you use Git, simply make sure that the file
+is covered by your `.gitignore` file.
+
+Otherwise, you cannot exclude files from REUSE compliance testing. It would go
+entirely against the purpose of REUSE: making sure that every file has a
+copyright and license tag.
+
+If you really want to exclude a file, consider using the
+[CC0](https://creativecommons.org/publicdomain/zero/1.0/) license for this
+file.  By doing this, you put the file in the public domain, or your country's
+equivalent.
+
+There is one exception: License files are automatically excluded from compliance
+testing.
+
+<!-- FIXME: Link to uncopyrightable question and DEP5 question -->
+
+## Do you support a version control system other than Git? {#no-git}
+
+Currently we do not, but please [get in
+touch](https://github.com/fsfe/reuse-tool/issues).
+
+# Licensing and compliance {#licensing}
+
+## Which licenses are compatible? {#compatible-licenses}
+
+## Is there a standard format for declaring copyright? {#standard-copyright}
+
+Copyright year copyright holder email
+
+## Which years do I include in the copyright statement? {#years-copyright}
+
+## Do I need to include both GPL-3.0-or-later and GPL-3.0-only in my repository? {#gpl-plus}
+
+Members of the GPL family are listed separately in the SPDX License List as
+-only and -or-later, even though the license texts are identical. If you have
+code under only one of these licenses, we recommend that you only include that
+one license.
+
+If you have code under both licenses, we recommend that you include both
+licenses separately.
```

Comment:
It might be a good idea to clarify what it means “under both licenses” here.

1. `GPL-2.0-or-later AND GPL-2.0-only` – i.e. there is some code under GPL-2.0-or-later and some code under GPL-2.0-only; vs
2. `GPL-2.0-or-later` – i.e. the user may decide to take the code either under GPL-2.0-only or GPL-3.0-only or GPL-3.0-or-later (or GPL-2.0-or-later), so the question is whether one needs to attach both version 2 and 3 of the GPL.

I think we are (or at least should be) talking about 1).

As for 2), I think we would eventually run into the problem of asking people to maintain their projects to attach new versions of `*GPL-*-or-later` and `*+` licenses to source code even when this new version did not exist yet, when they wrote the code.

Reply:
The reference was to (1.). I'll see if that can be made clearer.

(2.) is a difficult one. I don't think projects should include later versions of the GPL just because they have "or later" in there, but I can see why that might be an issue.